Extensions/dnSpy.Debugger/dnSpy.Debugger/Dialogs/AttachToProcess/AttachToProcessVM.cs

```diff
@@ -57,6 +57,19 @@ sealed class AttachToProcessVM : ViewModelBase, IGridViewColumnDescsProvider, IC
 
 		public string Title { get; }
 
+		/// <inheritdoc />
```

Comment:
XML doc comment not needed.

The getter should be on the same line as the property name: `HasError => hasError;`